### PR TITLE
New package: ETL-0.04

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/ETL/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'ETL'
+pkgname=ETL
+version=0.04
+revision=1
+_rev=19
```
Comment (xtraeme) [inline srcpkgs/ETL/template]: The real version is `0.04.19` not `0.04`, there's no reason to separate `${_rev}`.

Comment (diogoleal) [inline srcpkgs/ETL/template]: changed!

File: srcpkgs/ETL/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'ETL'
+pkgname=ETL
+version=0.04.19
+revision=1
+_version_synfig=1.0.1
+wrksrc=${pkgname}-${version}
```
Comment (beefcurtains) [inline srcpkgs/ETL/template]: This is pretty much the default.

Comment (diogoleal) [inline srcpkgs/ETL/template]: Ops! changed!

**Comment by beefcurtains** [comment]
Perhaps there should be a ``-devel``?

**Comment by diogoleal** [comment]
@beefcurtains  the ETL is a library development.  The reason I'm creating it template is that it is a synfig dependence.

**Comment by beefcurtains** [comment]
Then by definition it should have a -devel

**Comment by diogoleal** [comment]
-devel created!

File: srcpkgs/ETL/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
+:20:# Template file for 'ETL'
```
Comment (xtraeme) [inline srcpkgs/ETL/template]: The first 4 characters must be removed.

File: srcpkgs/ETL/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="GPL-3"
+homepage="http://www.synfig.org/"
+distfiles="http://download.tuxfamily.org/synfig/releases/${_version_synfig}/${pkgname}-${version}.tar.gz"
+checksum=21696881fab1b9a3458aaa77a8b10c899c297b2d132e3ed4cfb7ffa29cd8b8fe
+
+ETL-devel_package(){
+	short_desc+=" - development files"
+	pkg_install() {
+	vmove usr/lib/pkgconfig
```
Comment (xtraeme) [inline srcpkgs/ETL/template]: - Please respect the indentation with tabs
- Development pkg is missing `depends`

**Comment by xtraeme** [comment]
Hmm wait, this package does not install any library so that it's a development pkg already.

There's no need to create an additional subpackage, just remove `-devel`.

**Comment by beefcurtains** [comment]
Alright, sorry about that.

**Comment by diogoleal** [comment]
-devel section and caracteres '+:20:' removed.

File: srcpkgs/ETL-devel [hunk 5]
```diff
@@ -0,0 +1 @@
+ETL
```
Comment (xtraeme) [inline srcpkgs/ETL-devel]: The `ETL-devel` symlink is unnecessary.

File: srcpkgs/ETL/template [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'ETL'
+pkgname=ETL
+version=0.04.19
+revision=1
+_version_synfig=1.0.1
```
Comment (xtraeme) [inline srcpkgs/ETL/template]: This var is unnecessary...

**Comment by diogoleal** [comment]
variable _version_synfig=1.0.1 and symbolic link removed.

**Pull request merged**